app/frontend/components/pages/staking/stakingPage.js

```diff
@@ -0,0 +1,41 @@
+const {h} = require('preact')
```

Comment:
I miss there some button to be able to return back to AdaLite, perhaps, the `Staking` link in the navbar could become `Wallet` when you enter the staking page

Reply:
I think the back link would be more user-friendly. @martinmaricak what do you think about it (Link back from staking page)?

Reply:
I would leave `Subscribe` as the only action. User is able to navigate via clicking on the logo. The idea of improving the navigation sounds good, but what we actually want is to gather emails. So we would be able to reach out to a lot of users once we launch staking.